hack/update-aro-hcp-types.sh

```diff
@@ -11,4 +11,13 @@ go-jsonschema \
     /home/deads/workspaces/aro-hcp/src/github.com/Azure/ARO-HCP/config/config.schema.json \
     > pkg/apis/aro-hcp/config.go
 
-sed -i 's/package aro-hcp/package arohcpapi/' pkg/apis/aro-hcp/config.go
\ No newline at end of file
+go-jsonschema \
```

Comment:
I don't know if it will be necessary but I added some flags to this tool to make less shitty models: `--struct-name-from-title --minimal-names`

Reply:
> --struct-name-from-title

produced non-compling code due to name conflicts



> --minimal-names

ok